src/data/defaults.txt

```diff
@@ -2397,6 +2397,7 @@ user	_skateBuff2	false
 user	_skateBuff3	false
 user	_skateBuff4	false
 user	_skateBuff5	false
+user    _slimeVialsHarvested 0
```

Comment:
These fields should be separated by tabs, not spaces. (In case you're wondering about the test failure.)

Reply:
Aye, I've just fixed it and rerun the tests, should pass now. thanks.